src/components/ApiKeys.tsx

```diff
@@ -72,7 +73,11 @@ export class Row extends React.Component<RowProps> {
 
 		return (
 			<tr>
-				<td>{this.props.team && <TeamName teamId={this.props.team} iconSize={32} />}</td>
+				{this.props.globalTeamId ? (
+					''
```

Comment:
is there a better way to do nothing?

Reply:
`undefined`, `false` and `null` are printed in React as `''`.

So common React pattern for `{a ? <Some /> : ''}` is just `{a && <Some />}` (if `a` is truthy, show `<Some />`). Similar pattern is `{a || <Some />}` (if `a` is truthy, show `a` or fallback to `<Some />`)

Reply:
```typescript
{!this.props.globalTeamId && <td>{this.props.team && <TeamName teamId={this.props.team} iconSize={32} />}</td>}
```

If `globalTeamId` is falsy then show `<td>...`.